Tests/TestingApps/PaywallsTester/PaywallsTester/Views/AppContentView.swift

```diff
@@ -150,6 +154,17 @@ struct AppContentView: View {
         }
     }
 
+    private func presentPaywallViewController() {
+        let paywall = PaywallViewController(displayCloseButton: true)
+        paywall.modalPresentationStyle = .pageSheet
+
+        UIApplication.shared
+            .currentWindowScene?
+            .keyWindow?
+            .rootViewController?
+            .present(paywall, animated: true)
```

Comment:
This isn't great (ignoring `nil`s), but it's just a test app.

Reply:
I guess we could log something if it's nil? Just so it's not confusing in case something goes wrong.

Reply:
Done